CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ This is the log of notable changes to EAS CLI and related packages.
 
 ### 🛠 Breaking changes
 
+- Change default iOS image for projects with Expo SDK <= 41 (SDK 41 won't build with Xcode 12.5). ([#479](https://github.com/expo/eas-cli/pull/479) by [@dsokal](https://github.com/dsokal))
```

Comment:
it's not breaking change because the current default is still set to macos-catalina-10.15-xcode-12.4

Reply:
Fair enough